[2021-10-27 15:46:44,354] driver_util.py->__init__ line:31 [INFO] : 当前正在启动app,请稍后.....
[2021-10-27 15:46:45,369] remote_connection.py->__init__ line:117 [INFO] : Could not connect to port 4723 on host 127.0.0.1
[2021-10-27 15:46:45,369] remote_connection.py->__init__ line:136 [INFO] : Could not get IP address for host: 127.0.0.1
